experimental/emscripten/testdata/invoke.wat

```diff
@@ -0,0 +1,115 @@
+(module
```

Comment:
this will be deleted once we can  bump minor (and merge the experimental function into the normal emscripten package)